.travis.yml

```diff
@@ -150,16 +149,6 @@ install:
           export QT_API=pyqt;
       fi
 
-    # Reinstalling the required numpy version as it may be downgraded.
-    # TODO It may be removed when/if Sherpa becomes np version agnostic
-    # (maybe as soon as v4.8). See discussion in
-    # https://travis-ci.org/gammapy/gammapy/jobs/99867950#L639
-    - if $SHERPA; then
-          conda install --yes --channel astropy numpy=$NUMPY_VERSION iminuit;
-          conda install --yes --no-pin --channel https://conda.anaconda.org/cxc/channel/dev sherpa;
```

Comment:
For the fortran compiler, you should install `gfortran` via `apt-get`, as done here:
https://github.com/sherpa/sherpa/blob/master/.travis.yml#L16

This seems to work fine with Python and Numpy / f2py from conda.

The sherpa `python setup.py install` time is a few minutes ... should be OK for our travis-ci setup.